### PR TITLE
Add metric plots

### PR DESCRIPTION
**Pull request by linakrisztian**
* add plotting of metric curves
* remove not needed `README.md` created by smplib
* add additional parameter for metrics path `output_train_metrics_path` --> Note: requires https://github.com/mundialis/m.neural_network/pull/10

### CODE REVIEW

File: m.neural_network.train/m.neural_network.train.py [hunk 1]
```diff
@@ -145,8 +184,51 @@ def main():
...
+        train_metrics["valid_dataset_f1"],
+        "F1 score",
+        os.path.join(options["output_train_metrics_path"], "f1_score.png"),
+        ylim=[0, 1],
+    )
+
     grass.message(
-        f"Classification model is trained and saved to {kwargs['output_model_path']}.",
+        f"Classification model is trained and saved to {options['output_model_path']}\n"
+        f"Training metrics are saved here {options['output_train_metrics_path']}",
```
Comment (metzm) [inline m.neural_network.train/m.neural_network.train.py:231]:
```suggestion
        f"Training metrics are saved to {options['output_train_metrics_path']}",
```

File: m.neural_network.train/m.neural_network.train.py [hunk 2]
```diff
@@ -145,8 +184,51 @@ def main():
...
+        train_metrics["valid_dataset_iou"],
+        "Intersection over Union (IoU)",
+        os.path.join(options["output_train_metrics_path"], "iou.png"),
+        ylim=[0, 1],
+    )
+    # F1 curve
+    plot_curve(
+        train_metrics["epoch"],
+        train_metrics["train_dataset_f1"],
+        train_metrics["valid_dataset_f1"],
```
Comment (linakrisztian) [inline m.neural_network.train/m.neural_network.train.py:223]:
```suggestion
        train_metrics["train_dataset_f1_score"],
        train_metrics["valid_dataset_f1_score"],
```
